account_move_line_stock_info/views/stock_move_view.xml

```diff
@@ -18,17 +23,60 @@
         <field name="inherit_id" ref="stock.view_picking_form" />
         <field name="arch" type="xml">
             <xpath
-                expr="//notebook//form//field[@name='description_picking']/.."
+                expr="//notebook//tree//field[@name='description_picking']/.."
                 position="after"
             >
-                <group name="account_move_lines_grp" string="Journal Items">
-                    <field
-                        name="account_move_line_ids"
-                        readonly="1"
-                        nolabel="1"
-                        colspan="2"
-                    />
-                </group>
+                <form string="Stock Moves">
```

Comment:
@CarlosVForgeFlow why are you doing this? You are breaking the inheritance, and the account_move_line_ids should only be added as a field to the form view.

Reply:
The form view Stock Moves that this xpath inherited from has been eliminated in Odoo 17.0. I added it here for the view to be the same as before.

Reply:
How about showing them in the tree-view instead, with the many2many_tags widget (with optional="hide") plus a button to open them?